diff --git a/node_modules/@bcp-nextgen-dx-component-factory/core/src/context/theme-provider/native/Theme.provider.js b/node_modules/@bcp-nextgen-dx-component-factory/core/src/context/theme-provider/native/Theme.provider.js
index f6dcd74..2b6e33d 100644
--- a/node_modules/@bcp-nextgen-dx-component-factory/core/src/context/theme-provider/native/Theme.provider.js
+++ b/node_modules/@bcp-nextgen-dx-component-factory/core/src/context/theme-provider/native/Theme.provider.js
@@ -2,7 +2,6 @@ import { jsx as _jsx } from "react/jsx-runtime";
 import dayjs from 'dayjs';
 import React from 'react';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
-import { SafeAreaProvider, initialWindowMetrics } from 'react-native-safe-area-context';
 import { ThemeProvider as Provider } from 'styled-components/native';
 import '../../../core/date';
 import { PortalProvider } from '../../portal-provider';
@@ -14,7 +13,7 @@ export const ThemeProvider = ({ theme, children, locale, localeData, }) => {
         ...theme,
         locale: locale || theme.locale,
     }), [theme, locale]);
-    return (_jsx(SafeAreaProvider, { initialMetrics: initialWindowMetrics, children: _jsx(Provider, { theme: withLocale, children: _jsx(PortalProvider, { children: _jsx(GestureHandlerRootView, { style: {
+    return (_jsx(Provider, { theme: withLocale, children: _jsx(PortalProvider, { children: _jsx(GestureHandlerRootView, { style: {
                         flex: 1,
-                    }, children: children }) }) }) }));
+                    }, children: children }) }) }) );
 };
